src/_community/pluralkit.md

```diff
@@ -1,62 +1,53 @@
 ---
-title: Tupperbox
+title: PluralKit
+redirect_from:
+  - /community/tupperbox
 ---
 
 {% admonition %}
-**Note:** This document is specific to Discord. If you're not making use of our community spaces on Discord, Tupperbox
+**Note:** This document is specific to Discord. If you're not making use of our community spaces on Discord, PluralKit
 won't be available.
 {% endadmonition %}
 
-[Tupperbox](https://github.com/Runi-c/Tupperbox) is a Discord bot that exists to allow users to proxy their messages
-via Discord webhooks. This has many practical applications, especially given that Discord doesn't have any features
-that support the plural community, or any built-in mental health aids.
+[PluralKit](https://pluralkit.me) is a Discord bot that exists to allow users to proxy their messages via Discord 
+webhooks. This has many practical applications, especially given that Discord doesn't have any features that support 
+the plural community, or any built-in mental health aids.
 
-Tupperbox has many fun and interesting uses for all kinds of situations, but in Quilt community spaces it should only 
-be used for [plurality](#plurality) or self-identity purposes, or as a mental health aid. We do not allow users to make use of 
-Tupperbox for role-playing.
+PluralKit has all kinds of fun and interesting uses, but in Quilt community spaces it should only be used for 
+[plurality](#plurality) or self-identity purposes, or as a mental health aid. We do not allow users to make use of 
+PluralKit for role-playing.
 
-If you see users talking with the bot tag, they're talking through Tupperbox. You can react to any message sent this
+If you see users talking with the bot tag, they're talking through PluralKit. You can react to any message sent this
 way with the `question` emote (❓) for information on which Discord account the message belongs to.
 
 ## General Usage
 
-The primary function of Tupperbox is automatically proxying messages that people using it send, in order to make those
-messages appear to come from someone else. This is done by making use of Discord webhooks - which has the unfortunate
-side effect of Discord stating that the messages were sent by a bot. Rest assured that this isn't the case - while the
-messages were proxied by Tupperbox, they were written by another member of the community
+The primary function of PluralKit is automatically proxying messages that people using it send, in order to make 
+those messages appear to come from someone else. This is done by making use of Discord webhooks - which has the 
+unfortunate side effect of Discord stating that the messages were sent by a bot. Rest assured that this isn't the 
+case - while the messages were proxied by PluralKit, they were written by a real member of the community.
```

Comment:
Part of me wants to say that having an exclamation mark on the end of this would just feel more right